doc/source/rllib/doc_code/rllib_in_60s.py

```diff
@@ -7,7 +7,7 @@
     PPOConfig()
     .environment("Taxi-v3")
     .rollouts(num_rollout_workers=2)
-    .framework("tf2")
```

Comment:
changing the framework makes the test run faster since when using tf2 we also need to enable eager tracing to achieve the same speed.